### PR TITLE
fix typo indicated in #234

### PR DESCRIPTION
**Pull request by antoniosanct**
@lukasj
I wish this PR would be useful for the project.

Regards,
Antonio.

### CODE REVIEW

File: api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java [hunk 1]
```diff
@@ -144,7 +144,7 @@
  * that doesn't have {@link XmlRootElement}. But they can't show up in an
  * XML instance (because they don't have XML element names).
  *
- * <p><b>Example 2: XML Schema Susbstitution group support</b>
+ * <p><b>Example 2: XML Schema substitution group support</b>
```
Comment (lukasj) [inline api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java:147]: first letter should be kept in upper case

Comment (antoniosanct) [reply api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java]: Fixed. However, At first commit I copied same format than line 37. Check it out and say to me if it is necessary to fix too.

Regards,
Antonio.

**Review by lukasj — APPROVED** [review]

File: api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java (hunk 1 above)
Comment (lukasj) [reply api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java]: the best would be to bring in consistency accross all usages ;-)

**Comment by lukasj** [comment]
btw: same typo is there in XmlElementDecl - can you fix it all at once?

File: api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java (hunk 1 above)
Comment (antoniosanct) [reply api/src/main/java/jakarta/xml/bind/annotation/XmlElementRef.java]: Fixed in XmlElementDecl too!

**Review by lukasj — APPROVED** [review]

**Pull request merged**